README.md

```diff
@@ -35,6 +35,18 @@ Municipal data can be downloaded with a script:
 ./scripts/fetch-data
 ```
 
+Install the pre-commit hook that runs the lint script before any commit.
+(`pre-commit` is a python package that facilitates installing pre-commit
+hooks; thus it runs on the host machine, not Docker, and is not included
+in `requirements.txt`.)
+
+```console
+pip install pre-commit
```

Comment:
Consider adding the pre-commit installation to `scripts/bootstrap`, to streamline setup.

We should still talk about it in the README though, and mention the `git commit --no-verify` escape.

Reply:
[0a3b98a](https://github.com/azavea/iow-boundary-tool/pull/179/commits/0a3b98a99dc8837d5eca719e7b1e23fe0aff1f92)